docs/guides/setting-up-a-networked-meta-avatar.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: docs
+title: Setting up a Networked Meta Avatar
+description: This guide walks you through setting up a networked meta avatar.
+
+---
+
+# Setting up a Networked Meta Avatar
+
+This guide will demonstrate how to quickly load a Meta Avatar and network it to other machines using Normal.
+
+When you're finished, you'll end up with an application that will let you see and talk to other player's avatars:
+
+![](./setting-up-a-networked-meta-avatar/player-wave.png "Meta Avatar waving at the viewer") 
+
+## Setup your App for Meta Avatars
+
+Before your app can load avatar data from Meta's servers, it will need permission to do so.  Ensure that your app is [configured as required for Meta Avatar usage](https://developer.oculus.com/documentation/unity/meta-avatars-app-config/), and that you have set your Oculus Platform Settings to point at the right app IDs
+
+## Setting up your VR environment
+
+Start by creating a blank unity VR scene, with the camera properly tracked to the player's head.
+
+![](./setting-up-a-networked-meta-avatar/camera.png "Camera settings") 
+
+Create a game object and add a **Realtime** component to it.  Point your **Realtime** component at a **NormcoreAppSettings** scriptable object that has your app id in it.  Then, add a **RealtimeAvatarManager** component.
+
+![](./setting-up-a-networked-meta-avatar/realtime.png "Realtime Settings settings")
+
+For the avatar system to initalize properly, your scene will require a number of components in it (**OvrAvatarManager**, **AvatarLODManager**, **GPUSkinningConfiguraiton** and **OvrAvatarShaderManager**).  The easiest way get these is to install the Meta Avatar SDK Sample Scenes, and use one of the prefabs they provide as part of that.
+
+First install the sample scenes through the package manager.  If you don't see the Meta Avatar SDK, you may need to [add the Meta Avatar SDK to your package list manually.](https://developer.oculus.com/downloads/package/meta-avatars-sdk/)
```

Comment:
This link does not show me how to do add the meta avatars SDK to my package list manually?

Reply:
I'm now stuck at this point. I don't know how to get this package installed. This page has a download button which brings me to the asset store or to a UPM registry page. I don't want to go the asset store route as it's being deprecated, and the link to the meta UPM registry doesn't make it clear how I actually add this package or even what package to add.

That said, I don't think linking out to this page is the right solution anyway. If someone had skipped installing the examples from the initial dialog that came up, is there really no button in the Editor they can hit to bring that dialog back up? It seems crazy to me that if you skip that step, your only course of action is to go to the asset store or manually look up a UPM package bundle ID and add it to your project.